llvm/test/Transforms/PGOProfile/fix_entry_count_sampled.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: llvm-profdata merge %S/Inputs/fix_entry_count_sampled.proftext -o %t.profdata
```

Comment:
Since this input `.proftext` file is only used for this test, can we use `split-file` and inline it here?

https://llvm.org/docs/TestingGuide.html#extra-files

Reply:
These testing oddities were mimicking the approach taken in other nearby tests covering the same functionality.  In particular [fix_entry_count.ll](https://github.com/llvm/llvm-project/blob/main/llvm/test/Transforms/PGOProfile/fix_entry_count.ll) was used as a base.

In `llvm/test/Transforms/PGOProfile/` no other test makes use of split-file for proftext data, does this make it reasonable to follow this localized convention despite the proftext only being used by one test?

Reply:
I think `split-file` was introduced relatively recently and `fix_entry_count.ll` was last updated 2 years ago. More recent proftext tests do use `split-file`.

https://github.com/llvm/llvm-project/blob/78e026f845fb4d924673a9d534cc36cf7b55473c/llvm/test/Transforms/Coroutines/coro-pgo-setbranchweights.ll#L1-L6

I like to use `split-file` because otherwise we have lots of similar `.proftext` files in `Inputs/` and it is slightly inconvenient to find their uses. It also means only one file needs to be updated if we want to change the test.